django_camunda/dmn/utils.py

```diff
@@ -63,3 +65,57 @@ def parse_dmn(xml: str) -> IntrospectionResult:
     inputs = [process_input_var(input_var) for input_var in input_vars]
     outputs = dict((process_output_var(output_var) for output_var in output_vars))
     return IntrospectionResult(inputs=inputs, output=outputs)
+
+
+def extract_decision_definition_inputs(
+    xml: _Element, definition_id: str
+) -> List[DMNInputParameter]:
+    decision_table = xml.find(
+        f".//decision[@id='{definition_id}']", namespaces=xml.nsmap
+    )
```

Comment:
Not sure if the actual xml tree contains sensitive data, but might be better to escape the xpath query:

```python
xml.xpath("//decision[@id=$definition_id]", definition_id=definition_id)
```

Reply:
Good point :thinking: 
Annoying that XPath cannot deal with the default namespace :(